src/com/mendix/widget/GoogleMaps/components/__tests__/Map.spec.ts

```diff
@@ -0,0 +1,201 @@
+import { shallow } from "enzyme";
+import GoogleMap from "google-map-react";
+import { DOM, createElement } from "react";
+
+import { Map, MapProps } from "../Map";
+import { Marker } from "../Marker";
+
+import {
+    EventMock, GeocoderLocationType, GeocoderMock, GeocoderStatus, LatLngBoundsMock,
+    LatLngMock, MapsMock, MarkerMock, MockGoogle
+} from "../../../../../../../tests/mocks/GoogleMaps";
```

Comment:
A, indeed, you have to do it because of ts-loader [bug](https://github.com/TypeStrong/ts-loader/issues/213) There might be better suggestion in the discussion there